src/main/java/com/greengroupecommerce/controllers/StripeController.java

```diff
@@ -0,0 +1,43 @@
+package com.greengroupecommerce.controllers;
+
+import com.stripe.Stripe;
+import com.stripe.model.checkout.Session;
+import com.stripe.param.checkout.SessionCreateParams;
+
+import lombok.Value;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.view.RedirectView;
+
+@Controller
+@RequestMapping("/call-checkout")
+public class StripeController {
+
+    @PostMapping
+    public RedirectView createCheckoutSession() throws Exception {
+        @Value("${stripe.api.secretKey}")
+        final String secretKey;
+
+        Stripe.apiKey = secretKey;
+
+        String YOUR_DOMAIN = "http://localhost:8080";
+        SessionCreateParams params =
+                SessionCreateParams.builder()
+                        .setMode(SessionCreateParams.Mode.PAYMENT)
+                        .setSuccessUrl(YOUR_DOMAIN + "/confirmation")
+                        .setCancelUrl(YOUR_DOMAIN + "/home")
+                        .addLineItem(
+                                SessionCreateParams.LineItem.builder()
+                                        .setQuantity(1L)
+                                        // Provide the exact Price ID (for example, pr_1234) of the product you want to sell
+                                        .setPrice("price_1O2Po6AJzwVAVVDsohgTU5z1")
```

Comment:
should we tweak the Product table schema to have a column that stores these price IDs?

Reply:
Most likely. Wasn't going to increment the database version until we had all the products we want added to stripe.